EasyPost/packages.lock.json

```diff
@@ -7,6 +7,20 @@
         "requested": "[13.0.1, 14.0.0)",
         "resolved": "13.0.1",
         "contentHash": "ppPFpBcvxdsfUonNcvITKqLl3bqxWbDCZIzDWHzjpdAHRFfZe0Dw9HmA0+za13IdyrgJwpkDTDA9fHaxOrt20A=="
+      },
```

Comment:
question: why are these files continuing to get bumped on most PRs for this lib. The whole point of a lock file is they are locked so deps don't keep being transient 😉

Reply:
I usually block these, it's my IDE. I'll reign it in. For what it's worth, these are dev utilities, not something that affects the end-user